static/app/views/releases/detail/commitsAndFiles/filesChanged.spec.tsx

```diff
@@ -0,0 +1,142 @@
+import selectEvent from 'react-select-event';
+import {CommitAuthorFixture} from 'sentry-fixture/commitAuthor';
+import {ReleaseFixture} from 'sentry-fixture/release';
+import {ReleaseProjectFixture} from 'sentry-fixture/releaseProject';
+import {RepositoryFixture} from 'sentry-fixture/repository';
+
+import {initializeOrg} from 'sentry-test/initializeOrg';
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import RepositoryStore from 'sentry/stores/repositoryStore';
+import type {CommitFile, ReleaseProject} from 'sentry/types';
+import {ReleaseContext} from 'sentry/views/releases/detail';
+
+import FilesChanged from './filesChanged';
+
+function CommitFileFixture(params: Partial<CommitFile> = {}): CommitFile {
+  return {
+    id: '111222',
+    orgId: 1,
+    author: CommitAuthorFixture(),
+    commitMessage: 'feat(issues): Add alert (#1337)',
+    filename: 'static/app/components/alert.tsx',
+    type: 'M',
+    repoName: 'getsentry/sentry',
+    ...params,
+  };
+}
+
+describe('FilesChanged', () => {
+  const release = ReleaseFixture();
+  const project = ReleaseProjectFixture() as Required<ReleaseProject>;
+  const {routerProps, routerContext, organization} = initializeOrg({
+    router: {params: {release: release.version}},
+  });
+  const repos = [RepositoryFixture({integrationId: '1'})];
+
+  function renderComponent() {
+    return render(
+      <ReleaseContext.Provider
```

Comment:
this is usually provided by ReleasesDetailContainer, but that component was doing a ton of stuff